.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,2 +1,16 @@
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
+#  Copyright 2020 Google LLC
```

Comment:
* I used ["2020" since this file was created in 2020](https://github.com/GoogleCloudPlatform/bank-of-anthos/commits/47d77c1756e5ec9be3ef258bfbd7bd658e046af5/.mvn/wrapper/maven-wrapper.properties).
* The java-docs-samples repo includes license headers for .properties files ([see java-docs-samples example here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/spanner/r2dbc/src/main/resources/application.properties)).